### PR TITLE
feat: Remove ZeroMQ MessageBus capability

### LINKED ISSUE #10: [UART] Remove ZeroMQ MessageBus capability
see https://github.com/edgexfoundry/device-virtual-go/pull/325

### PR DESCRIPTION
**Pull request by lenny-intel**
closes #10

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **N/A**
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?) 
- [ ] I have opened a PR for the related docs change (if not, why?) **N/A**
  <link to docs PR>

## Testing Instructions
run `make tidy` 
Verify the github.com/pebbe/zmq4 module is not in the go.mod file
run `make build` w/o ZMQ libs install on system
Verify builds successfully
run `make tests`
Verify tests all pass
run `make docker`
Verify builds docker image successfully

### CODE REVIEW

**Review by jim-wang-intel — APPROVED** [review]

**Comment by lenny-intel** [comment]
@cloudxxx8 , @FelixTing , you are committers on this repo so need you one of you approvals.

File: Attribution.txt
Comment (lindseysimple) [inline Attribution.txt:1]: Should we remove `github.com/pelletier/go-toml` and `github.com/pebbe/zmq4` in Attribution.txt? Other changes all looks good to me, thanks Lenny!

Comment (lenny-intel) [reply Attribution.txt:1]: Yes! Good catch!

Comment (lenny-intel) [reply Attribution.txt:1]: Done

**Review by lindseysimple — APPROVED** [review]
LGTM

**Pull request merged**
